### DISCUSSION TITLE
学生証明が受理されない

### DISCUSSION THREAD

**Discussion by ghost** [community/community#54199] [GitHub Education]
### Hi everyone,

GitHub Education Teamさま

以下のようなリジェクトメールが届き、学生証明が受理されません。
プロフィール欄にはフルネームと略歴を、学生証明書には学校名を書いており、メールにて指摘されている内容はクリアしています。
GitHub Education Teamさまが求める成果物の具体例を示していただけますでしょうか。

You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.
The image you selected does not appear to contain your school name. You may include multiple documents in your image, so long as they are legible. For example, if your original document is not in English then you might take a photo of the original next to an English translation.

**Comment by samus-aran** [comment]
Hi @EnchanterWug , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
